mix.exs

```diff
@@ -129,6 +132,57 @@ defmodule Trento.MixProject do
     ]
   end
 
+  defp docs do
+    [
+      main: "readme",
+      logo: "priv/static/images/trento.svg",
+      extra_section: "GUIDES",
+      source_url: @source_url,
+      skip_undefined_reference_warnings_on: [
+        "CHANGELOG.md",
+        "docs/development/environment_variables.md",
```

Comment:
This docs point to actual configuration files, and we can only attach md to the docs, so I just put this to remove the warning. But the links don't work.
I didn't make them work at least...

Reply:
this is easily fixed by using absolute links in the md files

Reply:
done!